src/server.js

```diff
@@ -29,7 +31,10 @@ module.exports = function() {
 	var app = express()
 		.use(allRequests)
 		.use(index)
-		.use(express.static("client"));
+		.use(express.static("client"))
+		.engine("html", expressHandlebars({extname: ".html", helpers: require("../client/js/libs/handlebars")}))
+		.set("view engine", "html")
+		.set("views", path.join(__dirname, "..", "client"));
```

Comment:
We can change it, but it really isn't simplified. You can speifically just use fs to grab the file as we have done before, and throw it into handlebars, but I couldn't actually get it to work and it definitely isn't simpler. This also allows us more flexibility to handle more files if we want to.

@astorije We could move it to a common folder if you want, or we can move the ones I created into the server and keep them seperate (as we don't use either of the same ones yet) but I don't think it's that big of an issue.